specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1839,12 +1851,18 @@ components:
       - "man"
       - "vrouw"
       - "onbekend"
+    TypePersoon_enum:
+      type: "string"
+      description: ":\n* `ingeschrevenpersoon`\n* `kadasterpersoon` "
+      enum:
+      - "ingeschrevenpersoon"
+      - "kadasterpersoon"
```

Comment:
moet hier ook niet duidelijk worden of het een kadasterpersoon dan wel een ingeschreven persoon is?

Reply:
Ik heb beschrijvende tekst toegevoegd, maar ik weet niet zeker of dat is wat je bedoelt.

Reply:
ik zei het niet goed. Ik bedoel dat we 4 soorten personen kennen:
1. ingeschreven natuurlijk persoon
2. ingeschreven niet-natuurlijk persoon
3. kadaster natuurlijk persoon
4. kadaster niet-natuurlijk persoon

de beschrijving bij /beslagen noemt 3 typen persoon, gebundeld naar 2 soorten:
          1.  Beslaglegger is een Persoon die bekend is in de BRP als ingeschreven persoon
          2.  Beslaglegger is niet bekend als ingeschreven persoon in de BRP of de beslaglegger is een rechtspersoon

Hiermee wordt dus een in het HR ingeschreven niet-natuurlijk persoon (met rsin) geïdentificeerd met parameter kadasterpersoonidentificatie. Er is ook geen parameter rsin gedefinieerd. Is dit een bewuste keuze? Zo ja, dan is mijn opmerking niet terecht en mag je deze negeren.

Reply:
Ja dat is een bewuste keuze. De ingeschreven niet natuurlijke persoon opzoeken bij het HR en daarmee de BRK bevragen levert slechts 40% van de resultaten op. Omdat dat niet betrouwbaar genoeg is, doen we het (totdat de dekking met het HR significant beter is) met de niet natuurlijke personen uit de BRK. Ook al zijn de zoekmogelijkheden zeer beperkt en zijn de gegevens niet actueel.